packages/rum/src/types/session-replay-utils.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable unicorn/filename-case */
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why not following the file name convention?

Reply:
Done!